### PR TITLE
ERC20 cross-chain adaption

### PR DESCRIPTION
**Pull request by Dinonard**
Migrated docs from @Maar-io to astar-docs repo with small adjustments.

### CODE REVIEW

File: docs/xcm/building-with-xcm/erc20-cross-chain-compatible.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    - Follow instructions on how to [Generate XC20 Address](https://docs.astar.network/docs/xcm/integration/tools). If your asset_id is 17 (=0x11), your EVM address will be `0xffffffff00000000000000000000000000000011`
+
+### 3. Transfer xc asset ownership to the XcBurrito smart contract
+To enable **xcBurrito** contract to be able to mint/burn you need to call `setTeam()` extrinsic on pallet-assets. Configure the `issuer` and `admin` to be **xcBurrito**. This will enable the contract to issue wrapped **XC20** assets. Since the EVM contract has 24 bytes (H160), we need to transform EVM address into SS58 format to be able to call this extrinsic.
+![polkadot-js-ss-1](https://user-images.githubusercontent.com/34627453/182795050-f10dad6e-dbd9-4ac9-b526-d51a66503cb9.png)
+
+You can renounce ownership of the **XC20** by calling `transferOwnership()` extrinsic on pallet-assets. Note that after this, you won't be able to
+change `issuer` or `admin` anymore so make sure those are set correctly.
+
+![polkadot-js-ss-2](https://user-images.githubusercontent.com/34627453/182795097-d145073b-cdf0-4c28-916f-66abb5c68abc.png)
```
Comment (andabakus) [inline docs/xcm/building-with-xcm/erc20-cross-chain-compatible.md:75]: Other images in the docs are local in the [local img folder](https://github.com/AstarNetwork/astar-docs/tree/doc/erc20-xc-compatible/docs/xcm/building-with-xcm/img) so these two pics could be put likewise. Currently images are numbered 1, 2, 3, 4 but personally I find this very obscure, maybe rather have them named erc20-cross-chain-compatible-1.png and erc20-cross-chain-compatible-2.png 

I can do this to be honest I do not know hot to add files to your PR? Perhaps after you merge PR with another PR?

**Review by andabakus — COMMENTED** [review]
One comment about the pics

File: docs/xcm/building-with-xcm/erc20-cross-chain-compatible.md (hunk 1 above)
Comment (Dinonard) [reply docs/xcm/building-with-xcm/erc20-cross-chain-compatible.md:98]: You just need to push to the same branch I'm working on, that's it.

**Comment by Dinonard** [comment]
Added **XC20+** chapter.

**Review by andabakus — APPROVED** [review]
Approved all together with the local images added.

**Pull request merged**